package.json

```diff
@@ -36,7 +36,7 @@
     "eslint-config-prettier": "^8.10.0",
     "eslint-plugin-prettier": "^5.0.0",
     "npm-run-all": "^4.1.5",
-    "prettier": "^3.0.1"
+    "prettier": "^2.8.8"
```

Comment:
This version of `prettier` is required for compatibility with Jest inline snapshots.